crates/ef-testing/build.rs

```diff
@@ -0,0 +1,69 @@
+use std::{
+    fs,
+    path::{Path, PathBuf},
+    sync::Arc,
+};
+
+use build_utils::{converter::EfTests, dir_reader::DirReader, filter::Filter};
+use lazy_static::lazy_static;
+
+const SKIPPED_TESTS: &str = "../../blockchain-tests-skip.yml";
+const CACHED_SKIPPED_TESTS: &str = "../../tmp/blockchain-tests-skip.yml";
+
+lazy_static! {
+    static ref SUITE_PATH: PathBuf = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
+        .join("./ethereum-tests/BlockchainTests/GeneralStateTests");
+    static ref INTEGRATION_TESTS_PATH: PathBuf =
+        PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("./tests/");
+}
+
+fn main() {
+    // Tell Cargo that if the given file changes -> to rerun this build script.
+    println!("cargo:rerun-if-changed={SKIPPED_TESTS}");
```

Comment:
maybe "cleaner" to use the macro directly

`rerun_if_changed!(SKIPPED_TESTS)`

https://docs.rs/cargo-emit/latest/cargo_emit/macro.rerun_if_changed.html

Reply:
ah interesting, did not know you could use the macro